sys/linux/usbip.txt

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <drivers/usb/usbip/usbip_common.h>
+
+resource usbip_server_fd[fd]
```

Comment:
We have all fd subtypes start with `fd_` prefix, so this should be `fd_usbip_server`.